src/components/FieldLabel/FieldLabel.tsx

```diff
@@ -15,13 +20,35 @@ interface FieldLabelProps extends VibeComponentProps {
 
 const FieldLabel: FC<FieldLabelProps> = forwardRef(
   (
-    { icon = "", iconLabel = "", labelText = "", labelFor = "", iconClassName = "", labelClassName = "" },
+    {
+      icon = "",
+      descriptionIcon = undefined,
+      description = "",
+      iconLabel = "",
+      labelText = "",
+      labelFor = "",
+      iconClassName = "",
+      labelClassName = ""
+    },
     ref: ForwardedRef<HTMLLabelElement>
   ) => {
     if (!labelText) {
       return null;
     }
 
+    const renderIconWithToolTip = () => {
+      return (
+        <Tooltip content={description}>
+          <Icon
+            icon={descriptionIcon ?? Info}
```

Comment:
after setting Info as default you will be able to change this line to this
```suggestion
            icon={descriptionIcon}
```